optuna/trial.py

```diff
@@ -197,13 +204,28 @@ def suggest_loguniform(self, name, low, high):
             Suggest penalty parameter ``C`` of `SVC <https://scikit-learn.org/stable/modules/
             generated/sklearn.svm.SVC.html>`_.
 
-            .. code::
+            .. testcode::
+
+                import optuna
+                import numpy as np
+                from sklearn.svm import SVC
+                from sklearn.model_selection import train_test_split
 
-                >>> def objective(trial):
-                >>>     ...
-                >>>     c = trial.suggest_loguniform('c', 1e-5, 1e2)
-                >>>     clf = sklearn.svm.SVC(C=c)
-                >>>     ...
+                def objective(trial):
+                    c = trial.suggest_loguniform('c', 1e-5, 1e2)
+                    assert 1e-5 <= c < 1e2
```

Comment:
Please remove the assertion to simplify the example. It looks like a test code rather than an example because the range is inputted right before this line.

```suggestion
```